torch/csrc/autograd/saved_variable.cpp

```diff
@@ -39,8 +39,10 @@ SavedVariable::SavedVariable(
     // follow.
     TORCH_CHECK(
         !variable.is_inference(),
-        "Inference tensors cannot be saved for backward. To work around "
-        "you can make a clone to get a normal tensor and use it in autograd.")
+        "Inference tensors cannot be saved for backward. Please do not use "
+        "Tensors created in inference mode in computation tracked by autograd. "
+        "To work around this, you can make a clone to get a normal tensor and "
+        "use it in autograd.")
```

Comment:
Let's guide the user to something that's potentially a direct workaround?

```suggestion
        "use it in autograd. or use the `torch.no_grad` context manager during inference")
```